### ISSUE TITLE
A11y: [Continue to Element Pro?] App icon has no alternative text (announces file n...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-ios#5237]
- **Checkpoint ID:** 11.1.1.1b/2.1
- **View/Component:** Continue to Element Pro?

### Description
App icon has no alternative text (announces file name).

### Recommendation
Provide meaningful alt text or hide if decorative.

--- 
*Report Date: 20.02.2026*

**Comment by Velin92** [comment]
This is a webview, so we can't do a lot about it
